src/scripts/main.js

```diff
@@ -2,7 +2,31 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tr = document.querySelector('tr');
```

Comment:
Use 'document.querySelector('.dashboard')' instead of 'document.querySelector('tr')' to select the table with class 'dashboard'.